packages/wrangler/src/r2/index.ts

```diff
@@ -221,22 +221,30 @@ export function r2(r2Yargs: CommonYargsArgv) {
 				"create <name>",
 				"Create a new R2 bucket",
 				(yargs) => {
-					return yargs.positional("name", {
-						describe: "The name of the new bucket",
-						type: "string",
-						demandOption: true,
-					});
+					return yargs
+						.positional("name", {
+							describe: "The name of the new bucket",
+							type: "string",
+							demandOption: true,
+						})
+						.option("location", {
+							describe: "A hint to control where the bucket will be created",
+							alias: "l",
+							type: "string",
+							default: "auto",
+							choices: ["auto", "apac", "eeur", "enam", "weur", "wnam"],
```

Comment:
So as @isaac-mcfadyen suggested, I tried to explain what these abbreviations mean. I tried a couple of things:

1. I changed the `choices` to include the meanings of locations, and coupled that with the `coerce` option, but that forces the user to type in `apac (Asia-Pacific)` in its entirety, which is of course not ideal
2. I tried adding `( apac = Asia-Pacific, ... )` to the `describe` string above, which works but I don't like it very much.
3. Linking to https://developers.cloudflare.com/r2/buckets/data-location/#available-hints is also not ideal.

I can go ahead with `2.` unless people here have better ideas?

Reply:
Check this out - facing similar issues for D1 @sdnts: https://github.com/cloudflare/workers-sdk/pull/2997

Reply:
Yeah I think I'm going to commit 2. We should be consistent with what locations we show though: I see your PR expands "weur" to "Europe (West)". I'm going off of https://developers.cloudflare.com/r2/buckets/data-location/#location-hints